### PR TITLE
Group 6 postmortems

### PR DESCRIPTION
**Pull request by antlet**
We finished the rules in both sections of postmortems. Any changes are welcome.

### CODE REVIEW

**Review by corlettim — CHANGES_REQUESTED** [review]
Your work looks good! Make sure to update your branch with what is in the main branch.

**Review by munzekm — CHANGES_REQUESTED** [review]
Looks good! Make sure group members add names to the bottom of the conduct.md.

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Your additions look good! Please don't forget to resolve merge conflicts and add your names at the bottom in the appropriate format

**Review by corlettim — DISMISSED** [review]

File: conduct.md [hunk 1]
```diff
@@ -149,14 +149,20 @@ provide great insight in how to improve in future projects.
...
   * Members shall not
     * Insult others while presenting criticism.
     * Blame other members for any issues with the final product.
     * Refuse to collaborate in fixing remaining issues while a project or lab has not been completed.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for the postmortems.
+    * Single out specific team members when discussing the results of a project or a lab.
+    * Make fun of or belittle people's inputs/contributions.
+    * Be closed minded in regard to correct solutions.
+
+Andre Hance, Thomas Antle, Bailey Matrascia, Wesley Long
```
Comment (corlettim) [inline conduct.md]: I think you only need your names at the bottom so you can remove them from this spot

**Review by corlettim — CHANGES_REQUESTED** [review]

**Review by corlettim — APPROVED** [review]

**Review by munzekm — APPROVED** [review]

**Review by noorbuchi — APPROVED** [review]

**Pull request merged**